src/ServiceControl.Audit.Persistence.Tests.RavenDb5/EmbeddedLifecycleTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace ServiceControl.Audit.Persistence.Tests
+{
+    using System.IO;
+    using System.Linq;
+    using System.Net.NetworkInformation;
+    using System.Threading.Tasks;
+    using NUnit.Framework;
+
+    [TestFixture]
+    class EmbeddedLifecycleTests : PersistenceTestFixture
+    {
+        public override async Task Setup()
+        {
+            SetSettings = s =>
+            {
+                var dbPath = Path.Combine(TestContext.CurrentContext.WorkDirectory, "Tests", "Embedded");
+                var databaseMaintenancePort = FindAvailablePort(33335);
+
+                s.PersisterSpecificSettings["ServiceControl/Audit/RavenDb5/UseEmbeddedInstance"] = bool.TrueString;
+                s.PersisterSpecificSettings["ServiceControl.Audit/DbPath"] = dbPath;
+                s.PersisterSpecificSettings["ServiceControl.Audit/HostName"] = "localhost";
+                s.PersisterSpecificSettings["ServiceControl.Audit/DatabaseMaintenancePort"] = databaseMaintenancePort.ToString();
+            };
+
+            var instance = await SharedEmbeddedServer.GetInstance();
+
+            //make sure to stop the global instance first
+            instance.Dispose();
```

Comment:
Does the startup on other tests restart the instance?

Reply:
It should restart automatically